docs-2.0/nebula-exchange/use-exchange/ex-ug-import-from-clickhouse.md

```diff
@@ -12,31 +12,31 @@ This example is done on MacOS. Here is the environment configuration information
 
 - Hardware specifications:
   - CPU: 1.7 GHz Quad-Core Intel Core i7
-  - memory: 16 GB
+  - Memory: 16 GB
 
-- Spark: 2.4.7, Stand-alone
+- Spark: 2.4.7, stand-alone
 
-- Hadoop: 2.9.2, Pseudo-distributed deployment
+- Hadoop: 2.9.2, pseudo-distributed deployment
 
 - ClickHouse: docker deployment yandex/clickhouse-server tag: latest(2021.07.01)
 
-- Nebula Graph: {{nebula.release}} ([Deploy Nebula Graph with Docker Compose](../../4.deployment-and-installation/2.compile-and-install-nebula-graph/3.deploy-nebula-graph-with-docker-compose.md))
+- Nebula Graph: {{nebula.release}}. [Deploy Nebula Graph with Docker Compose](../../4.deployment-and-installation/2.compile-and-install-nebula-graph/3.deploy-nebula-graph-with-docker-compose.md).
 
 ## Prerequisites
 
 Before importing data, you need to confirm the following information:
 
 - Nebula Graph has been [installed](../../4.deployment-and-installation/2.compile-and-install-nebula-graph/2.install-nebula-graph-by-rpm-or-deb.md) and deployed with the following information:
 
-  - IP address and port of Graph and Meta services.
+  - IP addresses and ports of Graph and Meta services.
 
   - User name and password with Nebula Graph write permission.
```

Comment:
1. Add ”a/the“ at the beginning of the sentence. User here is not the User role in Nebula Graph, it indicates any username right?
2. Does Nebula Graph write permission indicate the write permission to Nebula Graph spaces?